content/100-getting-started/02-setup-prisma/200-add-to-existing-project/index.mdx

```diff
@@ -2,6 +2,15 @@
 title: 'Add to existing project'
 metaTitle: 'Add Prisma to an existing project'
 metaDescription: 'Learn how to add Prisma to an existing Node.js or TypeScript project by connecting it to your database of choice and generating a Prisma Client for database access.'
+toc: false
 ---
 
+<TopBlock>
+
+Include Prisma in an existing project with the following tutorials guide that explain some core concepts as it guides you through integrating Prisma into your workflow.
```

Comment:
```suggestion
Include Prisma in an existing project with the following documentation, which explains some core concepts as it guides you through integrating Prisma into your workflow.
```

Reply:
Thanks @janpio , we did need some sentence there. I made a grammar tweak.

Reply:
Note that I copy-pasted that from somewhere.